try {
  console.log("try");
  throw "error";
} catch {
  console.log("catch");
} finally {
  console.log("finally");
}
